### DISCUSSION TITLE
How to run the Swift example app?

### DISCUSSION THREAD

**Discussion by bbudron** [WalletConnect/walletconnect-monorepo#714] [Developer Support]
Im confused about how to get the example app up and running

**Comment by chadyj** [comment]
@bbudron What issue are you having?

**Comment by bspringston** [comment]
When opening ExampleApp.xcodeproj I get a recursive popup message "The file “project.xcworkspace” could not be unlocked." that I can't get past

**Reply by llbartekll** [reply]
could you check if it works for you on develop branch?

**Comment by bspringston** [comment]
I added the develop branch via SPM. Is there an example of how to use it? I’m not familiar with your code base can you point me in the right direction?

**Comment by llbartekll** [comment]
clone the repo git@github.com:WalletConnect/WalletConnectSwiftV2.git
there are two sample apps a "wallet" and a "dapp"

**Comment by bspringston** [comment]
I’m getting dependency build errors:
